fdbserver/VersionedBTree.actor.cpp

```diff
@@ -10128,6 +10128,9 @@ TEST_CASE("Lredwood/correctness/btree") {
 	state int maxColdStarts = params.getInt("maxColdStarts").orDefault(300);
 	// Max number of records in the BTree or the versioned written map to visit
 	state int64_t maxRecordsRead = params.getInt("maxRecordsRead").orDefault(300e6);
+	// Max test runtime (in seconds). After the test runs for this amount of time, the next iteration of the test
+	// loop will terminate.
+	state double maxRuntimeWallTime = params.getDouble("maxRuntimeWallTime").orDefault(20 * 60);
```

Comment:
This is a bit tricky, because 20 minutes is probably too long to be running in a non-sanitizer run (20 minutes might actually be the timeout, but that's also something configured externally).

We could improve this a bit by changing the logic here to choose a different timeout if we were running in a sanitizer, or maybe better still we could configure the timeouts for both via test parameters to account for possible changes in the external timeout condition

Reply:
or can we read the externally configured timeout here and choose this threshold to be 2/3 of that configured value?